VideoLocker/res/values/errors.xml

```diff
@@ -2,47 +2,71 @@
 <resources>
 
     <!-- Network error messages -->
+    <!-- Error shown when  user attempts an action but needs an internet connection -->
     <string name="network_not_connected">You are not connected to the Internet.</string>
+    <!-- Concise error shown when user attempts an action but needs an internet connection -->
     <string name="network_not_connected_short">Network not reachable</string>
-    <string name="network_panel_dialog_message_help1">Please check your internet connection </string>
-    <string name="network_panel_dialog_help">This video can\'t load at the moment </string>
+    <!-- Concise error shown when user attempts an action but needs an internet connection -->
     <string name="network_panel_slow_dialog_message1">Please check your internet connection</string>
```

Comment:
Remove the 1 at the end, since the 2nd version of this warning has been removed.